sdk/core/perf/test/CMakeLists.txt

```diff
@@ -49,6 +49,12 @@ add_executable (
     src/random_stream_test.cpp
 )
 
+if (MSVC)
+  # Disable warnings
+  # - C6326: Google comparisons 
+  target_compile_options(azure-perf-unit-test PUBLIC /wd6326)
```

Comment:
Can you point to an example of our source that triggered this warning?

Reply:
We have this warning disabling in azure-core-tests as well and other test packages,
MSCV warns you when we do something like `ASSERT_EQ(a, b)` where a is a size_t const and b is size_t as well. MSVC will say that you are just comparing two constants as `4 == 2` or something like that. Because ASSER_EQ macro grom gtest ends up checking something like if(VALUE == OTHER_VALUE) => if(4 == 2).